client/src/partials/cash/modals/receipt.modal.html

```diff
@@ -102,7 +102,13 @@
 </div>
 
 <div class="modal-footer">
-  <button  class="btn btn-success" ng-click="CashReceiptModalCtrl.cancel()">
-    {{ "FORM.BUTTONS.DISMISS" | translate }}
+  <button  class="pull-left btn btn-default" ng-click="CashReceiptModalCtrl.cancel()">
```

Comment:
As per #268, all buttons should be pushed to the right.  If you think this is a poor user experience, please comment on the issue and I'll reopen it for discussion!